demo/admin/src/common/apollo/inMemoryCache.ts

```diff
@@ -0,0 +1,12 @@
+import { InMemoryCache, PossibleTypesMap } from "@apollo/client";
+import fragmentTypes from "@src/fragmentTypes.json";
+
+const possibleTypes: PossibleTypesMap = {};
+for (const type of fragmentTypes.__schema.types) {
+    possibleTypes[type.name] = type.possibleTypes.map((possibleType) => possibleType.name);
+}
+
+export const inMemoryCache = new InMemoryCache({
```

Comment:
The cache should be part of the `createApolloClient` factory. Otherwise, two instances of the Apollo Client use the same cache, which doesn't make sense.